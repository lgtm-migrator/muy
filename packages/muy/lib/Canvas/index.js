import React from "react"
import Box from "@material-ui/core/Box"

const Canvas = ({ display, ...props }) => (
  <Box component={"canvas"} display={display} {...props} />
)

export default Canvas

Canvas.defaultProps = {
  display: "inline-block",
}
