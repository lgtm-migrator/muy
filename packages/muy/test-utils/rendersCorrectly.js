import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
