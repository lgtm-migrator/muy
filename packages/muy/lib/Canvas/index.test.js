import React from "react"
import Canvas from "."
import rendersCorrectly from "../../test-utils/rendersCorrectly"

describe("<Canvas /> component", () => {
  it("renders correctly", () => {
    rendersCorrectly(<Canvas />)
  })
})
