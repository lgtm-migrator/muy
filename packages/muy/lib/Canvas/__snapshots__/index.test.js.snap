// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Canvas /> component renders correctly 1`] = `
<canvas
  className="MuiBox-root MuiBox-root-1"
/>
`;
